xonsh/commands_cache.py

```diff
@@ -126,10 +125,12 @@ def lazy_locate_binary(self, name):
         if ON_WINDOWS:
             # Windows users expect to be able to execute files in the same
             # directory without `./`
-            local_bin = next((fn for fn in possibilities if os.path.isfile(fn)), None)
+            local_bin = next((fn for fn in possibilities if os.path.isfile(fn)),
+                             None)
             if local_bin:
-                return os.path.abspath(local_bin)                              
-        cached = next((cmd for cmd in possibilities if cmd in self._cmds_cache), None)
+                return os.path.abspath(local_bin)
+        cached = next((cmd for cmd in possibilities if cmd in self._cmds_cache),
+                      None)
         if cached:
```

Comment:
Well, I have just be using PEP8.  Flake8 is set to 180 right now, which is clearly too high.  If we are going to deviate from PEP8, then we need to document how and in what circumstances.